src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -265,8 +285,8 @@ class UploadDetachedFileValidation extends React.Component {
 		validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
 		if (!this.state.headerErrors && this.state.validationFinished) {
 			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published}/>;
-			if (this.state.published){
-				// Submission is published and cannot be re-published
+			validationButton = <button className='pull-right col-xs-3 us-da-button' onClick={this.openModal.bind(this)}>Publish</button>;
```

Comment:
What's the point of this default button if it's always going to be overwritten? If you want it to open the modal, shouldn't that be in the checkFabsPermissions if statement? As it is, we'll never actually see this modal.